week3_HM/todo/src/components/TodoList.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import input from './buttons'
+import InsertTasks from './insertTasks'
+//import DatePicker from 'react-datepicker';
+//import moment from 'moment';
+
+const data = require('../todo.json')
```

Comment:
Please use one module system, not both. You can easily accomplish the same thing here with:

    import data from '../todo.json'